usagov/vote-gov-drupal
# VOTE-1156: Update header and copy of state content type online instances

## Jira ticket

[VOTE-1156](https://cm-jira.usa.gov/browse/VOTE-1156)

## Description

Update text in header and update body copy of online registration template:
HEAD: "How to register and update your registration"
BODY: "Complete your online registration on [state]'s election website."

## Deployment and testing

### QA/Testing instructions

1. lando retune
2. Visit an online registration page (Alabama) and confirm header and body content is correct
3. Test that it translates

## Checklist for the Developer




## Checklist for the Peer Reviewers
